service/common/src/main/java/org/apache/polaris/service/admin/PolarisAdminService.java

```diff
@@ -664,6 +665,15 @@ private void validateUpdateCatalogDiffOrThrow(
             "Cannot modify ExternalId in storage config from %s to %s",
             currentStorageConfig, newStorageConfig);
       }
+
+      if ((currentAwsConfig.getUserARN() != null
```

Comment:
I could not find where `userArn` is used by the Polaris Server. Could you point me at the code where the configuration is used?